lib/project_types/script/layers/infrastructure/languages/other_project_creator.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Script
+  module Layers
+    module Infrastructure
+      module Languages
+        class OtherProjectCreator < ProjectCreator
+          def self.config_file
+            "script.config.yml"
+          end
+
+          def self.metadata_file
+            "metadata.json"
+          end
+
+          def setup_dependencies
+            generate_config
+            generate_metadata
+          end
+
+          def create_start_message
+            ctx.message(
+              "script.create.preparing_project",
+              sparse_checkout_repo,
+              project_name
+            )
+          end
+
+          def create_inprogress_message
+            ctx.message(
+              "script.create.creating_other",
+              sparse_checkout_repo,
+              project_name
+            )
+          end
+
+          def create_finished_message
+            ctx.message("script.create.created_other", project_name)
+          end
+
+          private
+
+          # the config is equivalent to TS's script.config.yml
+          # ex: https://github.com/Shopify/scripts-apis-examples/blob/master/checkout/typescript/payment-methods/default/script.config.yml
```

Comment:
I know we chatted briefly about this Alex, but I'll ask it here in case others have opinions on this 😛 

Since we're effectively bootstrapping a new project with some template files, I wonder if it makes sense to reuse the `scripts-apis-examples` repo. The template for "other" projects could live in `/checkout/other/{apiName}/default` and could be fetched with sparse checkout.

Reply:
Yes I think this makes sense. I've gotten a few questions along the lines of "we should just be using sparse checkout here right?" so it seems the consensus is there. 

We probably don't actually need this hardcoded version as a temporary solution while we decide where to place the files... I'll actually just go ahead and whip up a branch on the examples repo where we can use this.

Reply:
https://github.com/Shopify/scripts-apis-examples/commit/8c0633ac416414d7cb40f4a7c5bf92285013c6e7